config/routes.rb

```diff
@@ -20,4 +20,7 @@
       get :star
     end
   end
+
+  get '/user_preferences', to: 'users#edit'
```

Comment:
Not a huge fan of `user_preferences` as a url. What do you think about changing it to `settings`?